src/components/CreateTaskFloatingButton.tsx

```diff
@@ -0,0 +1,25 @@
+import { Plus } from "lucide-react";
+import { useState } from "react";
+import { createPortal } from "react-dom";
+import TaskForm from "./TaskForm";
+
+export default function CreateTaskFloatingButton() {
```

Comment:
I would suggest you separate the button from the form, maybe you can extract the logic here and move this to it's parent level that must be a better approach for handling this